app/views/embeddable/multiple_choices/_show_choice.html.haml

```diff
@@ -0,0 +1,6 @@
+%div{:style => (embeddable.layout == Embeddable::MultipleChoice::LAYOUT_HORIZONTAL ? "display: inline-block; margin-right: 10px;" : "") }
```

Comment:
@sagartalla `horizontal` should be added as a class, and the styles should be put into a separate css file.
probably best to add this class to the `.choice-container` above.